File: instructor/06_colecciones/03_set.py
provincias={'Pichincha','Guayas'}
print('Crear Provincias -> ',provincias)

#print('Primera Provincia -> ',provincias[0])

#provincias[0]='PICHINCHA'

"""
for provincia in provincias :
    if provincia=='Pichincha':
        print('Cambio Nombre')
        provincia='PICHINCHA1'    
"""

#print('Cambio Primera Provincia -> ',provincias[0])

provincias.add('Azuay')
provincias.add('Azuay')
provincias.add('Azuay')
print('Nueva Provincia -> ',provincias)

print('Numero Provincias',len(provincias))

provincias.remove('Guayas')
print('Quitar Provincia')
print('Numero Provincias',len(provincias))
print('Provincias Final -> ',provincias)

print('Recorrer Provincias')
for provincia in provincias :
    print('Provincia =', provincia)

File: instructor/99_js/06_colecciones/03_set.js
let provincias=new Set(['Pichincha','Guayas'])
console.log('Crear Provincias -> ',provincias)

//console.log('Primera Provincia -> ',provincias[0])

//provincias[0]='PICHINCHA'

/*
for provincia in provincias :
    if provincia=='Pichincha':
        console.log('Cambio Nombre')
        provincia='PICHINCHA1'    
*/

//console.log('Cambio Primera Provincia -> ',provincias[0])

provincias.add('Azuay')
provincias.add('Azuay')
provincias.add('Azuay')
console.log('Nueva Provincia -> ',provincias)

console.log('Numero Provincias',provincias.lenght)

provincias.delete('Guayas')
console.log('Quitar Provincia')
console.log('Numero Provincias',provincias.lenght)
console.log('Provincias Final -> ',provincias)

console.log('Recorrer Provincias')
for(provincia of provincias)
    console.log('Provincia =', provincia)